create_aio_app/main.py

```diff
@@ -1,25 +1,25 @@
+import os
+
+from cookiecutter.main import cookiecutter
+
 from create_aio_app.utils.config import parse_arguments
-from create_aio_app.utils.generator import (
-    copy_template,
-    rename_dirs,
-    render_project_template,
-    remove_unnecessary_directories,
-)
 
 
 def main():
-    '''
-
-    :return:
-    '''
     args = parse_arguments()
+    template_path = f'{os.path.dirname(os.path.abspath(__file__))}/template/'
 
-    copy_template(args['name'])
-    rename_dirs(args['name'])
-    remove_unnecessary_directories(args)
+    if not args.get('name'):
+        cookiecutter(template_path)
+    else:
+        ctx = {
```

Comment:
Can't cookiecutter's CLI do the same?

Reply:
```
create-aio-app test --wuthout-postgres

vs

cookiecutter --no-input gh:aio-libs/create-aio-app/ project_name=test user_postgres=n
```

Reply:
I just want to simplify your job: you're going to have to **maintain** this. Maintaining things is not an easy job. The less routine you have — the more time you get for doing actual fun stuff :)

Also, users of cookiecutter are used to this. Why force them to learn another 10 ways of doing the same? It's against the Zen.

If you want to pre-fill template arg it's fine, but having multiple *different* CLI interfaces is questionable.